pkgs/tools/admin/balena-cli/default.nix

```
@@ -0,0 +1,66 @@
+
+{
+ stdenv
+, lib
+, fetchurl
+, unzip
+, installShellFiles
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${system}";
+
+  plat = {
+    x86_64-linux = "linux-x64";
+    x86_64-darwin = "macOS-x64";
+    x86_64-windows = "windows-x64";
+  }.${system} or throwSystem;
+
+  sha256 = {
+    x86_64-linux = "0pijb8p0rm681pmxsmrn0ybixyphwkaswabvji06s2y8kr55vxq9";
+    x86_64-darwin = "1y1s4hsdhki14jsfibhh56z47104id465k3a7fsx74n0ijygijb4";
+    x86_64-windows = "02x93zg5093mx4in80pgh3pbkrmwj9rmj7i85flfl0450zlxwaws";
+  }.${system} or throwSystem;
+in
+  stdenv.mkDerivation rec {
+    pname = "balena-cli";
+    version = "14.3.1";
+
+    src = fetchurl {
+      url = "https://github.com/balena-io/balena-cli/releases/download/v${version}/balena-cli-v${version}-${plat}-standalone.zip";
+      inherit sha256;
+    };
+
+    nativeBuildInputs = [ unzip ];
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/bin
+      cp -r ./* $out/
+
+      ln -s $out/balena $out/bin/balena
+
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      mainProgram = "balena";
+      description = "The official balena Command Line Interface";
+      longDescription = ''
+        The balena CLI is a Command Line Interface for balenaCloud or openBalena. It is a software
+        tool available for Windows, macOS and Linux, used through a command prompt / terminal window.
+        It can be used interactively or invoked in scripts. The balena CLI builds on the balena API
+        and the balena SDK, and can also be directly imported in Node.js applications. The balena CLI
+        is an open-source project on GitHub, and your contribution is also welcome!
+      '';
+
+      homepage = "https://github.com/balena-io/balena-cli";
+      changelog = "https://github.com/balena-io/balena-cli/blob/v${version}/CHANGELOG.md";
+
+      license = licenses.asl20;
+      maintainers = [ maintainers.kalebpace ];
+      platforms = platforms.all;
```

```suggestion
      platforms = with platforms; linux ++ darwin;
```

Indeed we should restrict even more, but I don't remember the exact expression for this.

---

Thanks for catching this